From c99d02bc23e2e66466491aa930bfa519e1ff43d0 Mon Sep 17 00:00:00 2001
From: Yurii Rashkovskii <me@yrashk.com>
Date: Wed, 14 Feb 2018 17:20:29 +0700
Subject: [PATCH] Problem: sit-web console warning

It's warning that it can't find the "iron-flex-alignmen"
style.

Solution: fix the typo in the name of the style.
---
 sit-web/webapp/src/dashboard-page/issue.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sit-web/webapp/src/dashboard-page/issue.html b/sit-web/webapp/src/dashboard-page/issue.html
index 6d7e242..f361a13 100644
--- a/sit-web/webapp/src/dashboard-page/issue.html
+++ b/sit-web/webapp/src/dashboard-page/issue.html
@@ -9,7 +9,7 @@
 <dom-module id="sit-issue">
     <template>
         <style include="issue-style"></style>
-        <style is="custom-style" include="iron-flex iron-flex-alignmen"></style>
+        <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
 
         <paper-card id="card" heading="[[ issue.summary ]]">
             <div class="card-content">
-- 
2.15.0

